common/protob/messages-tron.proto

```
@@ -151,6 +157,7 @@
  message TronSignMessage {
     repeated uint32 address_n = 1;                          // BIP-32 path to derive the key from master node
     required bytes message = 2;                             // message to be signed
```

Buf (1.55.1)

Field named "message" should not be required.

(FIELD_NOT_REQUIRED)